[2024-10-28T00:45:38.902+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-28T00:45:38.920+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: indices_argentina_v08.load scheduled__2024-10-27T00:00:00+00:00 [queued]>
[2024-10-28T00:45:38.930+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: indices_argentina_v08.load scheduled__2024-10-27T00:00:00+00:00 [queued]>
[2024-10-28T00:45:38.930+0000] {taskinstance.py:2865} INFO - Starting attempt 8 of 13
[2024-10-28T00:45:38.947+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): load> on 2024-10-27 00:00:00+00:00
[2024-10-28T00:45:38.956+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'indices_argentina_v08', 'load', 'scheduled__2024-10-27T00:00:00+00:00', '--job-id', '233', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpz267n7e9']
[2024-10-28T00:45:38.958+0000] {standard_task_runner.py:105} INFO - Job 233: Subtask load
[2024-10-28T00:45:38.960+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2705) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-28T00:45:38.960+0000] {standard_task_runner.py:72} INFO - Started process 2706 to run task
[2024-10-28T00:45:39.005+0000] {task_command.py:467} INFO - Running <TaskInstance: indices_argentina_v08.load scheduled__2024-10-27T00:00:00+00:00 [running]> on host 276004bbea2b
[2024-10-28T00:45:39.111+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='rolo' AIRFLOW_CTX_DAG_ID='indices_argentina_v08' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-10-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='8' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-27T00:00:00+00:00'
[2024-10-28T00:45:39.112+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-28T00:45:53.115+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-28T00:45:53.131+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-28T00:45:53.132+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=indices_argentina_v08, task_id=load, run_id=scheduled__2024-10-27T00:00:00+00:00, execution_date=20241027T000000, start_date=20241028T004538, end_date=20241028T004553
[2024-10-28T00:45:53.204+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-28T00:45:53.222+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-28T00:45:53.224+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
